.github/workflows/test-examples.yml

```diff
@@ -213,13 +213,40 @@ jobs:
           aws-role-to-assume: ${{ secrets.AWS_CI_ROLE_ARN }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Set up Minikube
+        uses: manusa/actions-setup-minikube@v2.13.0
+        with:
+          minikube version: 'v1.34.0'
+          kubernetes version: 'v1.29'
```

Comment:
Is there a reason to use v1.29? Could we drop this and use the default k8s version used by the configured minikube version?

Reply:
I just pinned it to what kind was using, but yes I am all for rolling forward

Reply:
Looks like `kubernetes version` is required, so I pinned it to the version here: https://github.com/manusa/actions-setup-minikube?tab=readme-ov-file#basic